playbooks/byo/openshift-cluster/initialize_groups.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Create initial host groups for localhost
+  hosts: localhost
+  connection: local
+  become: no
+  gather_facts: no
+  tags:
+  - always
+  tasks:
+  - include_vars: cluster_hosts.yml
+  - name: Evaluate group l_oo_all_hosts
+    add_host:
+      name: "{{ item }}"
+      groups: l_oo_all_hosts
```

Comment:
@sdodson, thanks for the explanation.  The irony of it is, if its internal to a role, then why are we in a playbook defining it.  At that point it becomes global.   Its fine for now but I'd prefer if we moved away from `l_`.  I'd prefer `group_`  or something that denotes our intention.